src/Microsoft.Azure.WebJobs.Host/Indexers/MethodInfoExtensions.cs

```diff
@@ -16,7 +16,9 @@ public static string GetFullName(this MethodInfo methodInfo)
                 throw new ArgumentNullException("methodInfo");
             }
 
-            return String.Format(CultureInfo.InvariantCulture, "{0}.{1}", methodInfo.DeclaringType.FullName, methodInfo.Name);
+            return TryGetCustomName(methodInfo, out string customName)
```

Comment:
isn't it necessary to declare `customName` before you pass it as an out parameter? I think the appveyor errors are because of this, though I'd love this shorthand